armadillo/src/main/java/org/molgenis/armadillo/controller/ObjectRequestBody.java

```diff
@@ -0,0 +1,5 @@
+package org.molgenis.armadillo.controller;
+
+import javax.validation.constraints.NotNull;
+
+public record ObjectRequestBody(@NotNull String name) {}
```

Comment:
Why use here a record why elsewhere we use @AutoValue?

Reply:
See discussion on Slack